website/docs/reference/project-configs/quoting.md

```diff
@@ -0,0 +1,154 @@
+---
+datatype: boolean # -ish, it's actually a dictionary of bools
+default: true
+---
+
```

Comment:
@drewbanin — I've always been a little hazy on quoting, would you mind reading through this?

Reply:
this is great as-is!